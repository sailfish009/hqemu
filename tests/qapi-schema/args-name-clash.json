# C member name collision
# FIXME - This parses, but fails to compile, because the C struct is given
# two 'a_b' members.  Either reject this at parse time, or munge the C names
# to avoid the collision.
{ 'command': 'oops', 'data': { 'a-b': 'str', 'a_b': 'str' } }
