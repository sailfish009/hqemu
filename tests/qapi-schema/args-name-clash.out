object :empty
object :obj-oops-arg
    member a-b: str optional=False
    member a_b: str optional=False
command oops :obj-oops-arg -> None
   gen=True success_response=True
